packages/contentstack-export-to-csv/package.json

```diff
@@ -1,28 +1,29 @@
 {
   "name": "@contentstack/cli-cm-export-to-csv",
   "description": "Export entities to csv",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "author": "Abhinav Gupta @abhinav-from-contentstack",
   "bugs": "https://github.com/contentstack/cli/issues",
   "dependencies": {
     "@contentstack/cli-command": "^1.1.0",
```

Comment:
@aman19K cli-command is at it's latest verion 1.2.0, why you are still using 1.1.0?

Reply:
@netrajpatel version updated.